cppcache/include/geode/internal/chrono/duration.hpp

```diff
@@ -35,7 +35,7 @@ namespace duration {
 
 template <class Period>
 struct _suffix {
-  static constexpr char const* value = nullptr;
+  static constexpr char const* value = "<<unknown units>>";
```

Comment:
little weird that we can't know the units, but assuming this is the best we can do for now

Reply:
This is the default, anything that comes in using a ratio value we understand gets a "real" suffix.  The point of the change is to stop using nullptr, because "bad" things happen :)